Resources/Prototypes/Entities/Objects/Consumable/Food/ingredients.yml

```diff
@@ -463,6 +463,40 @@
         - ReagentId: Nutriment
           Quantity: 5
 
+- type: entity
+  name: tofu
+  parent: FoodBakingBase
+  id: FoodTofu
+  description: Solid white block with a subtle flavor.
+  components:
+  - type: FlavorProfile
+    flavors:
+      - tofu
+  - type: Sprite
+    state: tofu
+  - type: SolutionContainerManager
+    solutions:
+      food:
+        maxVol: 6
+        reagents:
+        - ReagentId: Nutriment
+          Quantity: 5
+  - type: SliceableFood
+    count: 3
+    slice: FoodTofuSlice
+
+- type: entity
+  name: tofu slice
+  parent: FoodBakingBase
+  id: FoodTofuSlice
```

Comment:
Slices should have food solution, a third of the big block, so if going by my aforementioned suggestion: 3 Protein, 2 Nutriment.